adm_program/languages/en.xml

```diff
@@ -1175,6 +1175,7 @@
   <string name="SYS_REMEMBER_ME">Stay signed in</string>
   <string name="SYS_REMOVE">Remove</string>
   <string name="SYS_REMOVE_MESSAGE">Remove message</string>
+  <string name="SYS_REORDER">Move #VAR1#</string>
```

Comment:
We should use the english translation for the string name. So you know within the code better what you mean. Here SYS_MOVE_VAR would be a better name.